spec/client_spec.rb

```diff
@@ -266,7 +301,9 @@ def redis.with
 
         expect {
           lock_manager.lock(resource_key, ttl)
-        }.to raise_error(RedisClient::CannotConnectError)
+        }.to raise_error do |e|
```

Comment:
let's also assert what error is expected:

```suggestion
        }.to raise_error(Redlock::LockAcquisitionError) do |e|
```